src/Page/Simulator.elm

```diff
@@ -148,7 +148,7 @@ init maybeQuery session =
       , query = query
       , displayMode = SimpleMode
       , modal = NoModal
-      , customCountryMixInputs = { fabric = Nothing, dyeing = Nothing, making = Nothing }
+      , customCountryMixInputs = query.customCountryMixes |> toCustomCountryMixFormInputs
```

Comment:
Nit: maybe we should just avoid piping as it's just a one arg fn call? `toCustomCountryMixFormInputs query.customCountryMixes `